### ISSUE TITLE
Lot's of read -1 errors dumped during aishell CTC training

### ISSUE THREAD

**Issue by robinn37** [athena-team/athena#92]
There were quite a lot of read errors dumped during the training process. However, it seemed no impact to the training. Not sure if you ever got such annoying errors. Anything wrong with my training set?

....
[1,7]<stderr>:INFO:absl:perform batch_wise_shuffle with batch_size 16
[1,2]<stderr>:INFO:absl:please be patient, enable tf.function, it takes time ...
[1,0]<stderr>:INFO:absl:please be patient, enable tf.function, it takes time ...
[1,7]<stderr>:INFO:absl:please be patient, enable tf.function, it takes time ...
[1,7]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,3]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,0]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,2]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,5]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,6]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,1]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,4]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,7]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,5]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,2]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,3]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,6]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,0]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,4]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,1]<stderr>:WARNING:absl:the length of logits is shorter than that of labels
[1,0]<stderr>:[373f86b536f2:00117] Read -1, expected 5393, errno = 1
[1,0]<stderr>:[373f86b536f2:00117] Read -1, expected 6345, errno = 1
[1,0]<stderr>:[373f86b536f2:00117] Read -1, expected 5873, errno = 1
[1,6]<stderr>:[373f86b536f2:00123] Read -1, expected 5184, errno = 1
[1,4]<stderr>:[373f86b536f2:00121] Read -1, expected 5184, errno = 1
[1,5]<stderr>:[373f86b536f2:00122] Read -1, expected 5440, errno = 1
[1,0]<stderr>:[373f86b536f2:00117] Read -1, expected 4864, errno = 1
[1,7]<stderr>:[373f86b536f2:00124] Read -1, expected 4928, errno = 1
[1,4]<stderr>:[373f86b536f2:00121] Read -1, expected 1002048, errno = 1
[1,7]<stderr>:[373f86b536f2:00124] Read -1, expected 1002048, errno = 1
[1,0]<stderr>:[373f86b536f2:00117] Read -1, expected 1002048, errno = 1
[1,1]<stderr>:[373f86b536f2:00118] Read -1, expected 1002048, errno = 1
[1,5]<stderr>:[373f86b536f2:00122] Read -1, expected 1002048, errno = 1
[1,6]<stderr>:[373f86b536f2:00123] Read -1, expected 1002048, errno = 1
[1,3]<stderr>:[373f86b536f2:00120] Read -1, expected 1002048, errno = 1
[1,2]<stderr>:[373f86b536f2:00119] Read -1, expected 1002048, errno = 1
[1,7]<stderr>:[373f86b536f2:00124] Read -1, expected 1002048, errno = 1
[1,4]<stderr>:[373f86b536f2:00121] Read -1, expected 1002048, errno = 1
[1,5]<stderr>:[373f86b536f2:00122] Read -1, expected 1002048, errno = 1
[1,6]<stderr>:[373f86b536f2:00123] Read -1, expected 1002048, errno = 1
[1,0]<stderr>:[373f86b536f2:00117] Read -1, expected 1002048, errno = 1
[1,3]<stderr>:[373f86b536f2:00120] Read -1, expected 1002048, errno = 1
[1,1]<stderr>:[373f86b536f2:00118] Read -1, expected 1002048, errno = 1
[1,2]<stderr>:[373f86b536f2:00119] Read -1, expected 1002048, errno = 1
[1,3]<stderr>:[373f86b536f2:00120] Read -1, expected 1002048, errno = 1
[1,4]<stderr>:[373f86b536f2:00121] Read -1, expected 1002048, errno = 1
[1,7]<stderr>:[373f86b536f2:00124] Read -1, expected 131072, errno = 1
[1,7]<stderr>:[373f86b536f2:00124] Read -1, expected 20480, errno = 1
[1,0]<stderr>:INFO:absl:global_steps: 38522     learning_rate: 2.2517e-04       loss: 1.5734    CTCAccuracy: 0.9167     Accuracy: 0.9323
[1,7]<stderr>:[373f86b536f2:00124] Read -1, expected 5248, errno = 1
...

**Comment by Some-random** [comment]
We haven't seen errors like this. Is this some kind of error message from Athena Transform? @aishinchi

**Comment by Some-random** [comment]
I did some searching on Google and this [issue](https://github.com/horovod/horovod/issues/503) pops out. Seems like those messages are typical in Docker and considered harmless.

**Comment by robinn37** [comment]
Thanks for the information.
